### PR TITLE
[quant][fx] Move all binary op configs to backend_config_dict

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75378
* #75377
* #75318
* #75317
* #75316
* #75315
* #75314
* #75260
* #75259
* #75258
* __->__ #75241

Summary:
We have a previous PR that enabled operator.add in backend_config_dict, this
PR moved the rest binary ops to backend_config_dict.
There are some ops left, which are not needed (previously fp16 ops), we
will move them in the following PR

Test Plan:
python test/test_quantization.py TestQuantizeFx
python test/test_quantization.py TestQuantizeFxOps
 python test/test_quantization.py TestFXNumericSuiteCoreAPIs

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D35403589](https://our.internmc.facebook.com/intern/diff/D35403589)

### CODE REVIEW

File: test/quantization/fx/test_quantize_fx.py [hunk 1]
```diff
@@ -5000,7 +5000,7 @@ def forward(self, x):
 
         m = M()
         expected_node_occurrence = {
-            ns.call_module(torch.ao.quantization.FusedMovingAvgObsFakeQuantize): 6,
+            ns.call_module(torch.ao.quantization.FusedMovingAvgObsFakeQuantize): 5,
```
Comment (vkuzo) [inline test/quantization/fx/test_quantize_fx.py:5003]: is this expected?

**Review by vkuzo — APPROVED** [review]

File: torch/ao/quantization/fx/quantization_patterns.py [hunk 2]
```diff
@@ -117,23 +117,10 @@ def is_standalone_module(self):
         return self.is_standalone_module_
 
 @register_quant_pattern(operator.sub)
```
Comment (andrewor14) [inline torch/ao/quantization/fx/quantization_patterns.py:119]: Will these be moved in this PR as well?

**Review by andrewor14 — APPROVED** [review]

File: test/quantization/fx/test_quantize_fx.py (hunk 1 above)
Comment (jerryzh168) [reply test/quantization/fx/test_quantize_fx.py:5003]: yeah, this PR also added support for all variants of add - torch.relu fusion, which exist partially before for (operator.add, torch.relu). This PR also supported (torch.add, torch.relu)

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35403589).

**Comment by jerryzh168** [comment]
@jerryzh168 has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35403589).

**Pull request closed without merging**